test/src/scene/instanceID.cpp

```diff
@@ -0,0 +1,80 @@
+#include "lidars.hpp"
+#include "scenes.hpp"
+#include "scene/Scene.hpp"
```

Comment:
We should only use API headers in the tests if possible. It seems that it was included because of `DEFAULT_ENTITY_ID` variable. When my previous comment would be resolved this include can be removed.

Reply:
in this case, the best I can do is to delete "scene/Scene.hpp"